### PR TITLE
Fix gps_rescue_min_start_dist

### PR DESCRIPTION
**Pull request by haslinghuis**
Sets `gps_rescue_min_start_dist` range to start at 20 in UI.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by KarateBrot — APPROVED** [review]

**Review by sugaarK — APPROVED** [review]

File: src/tabs/failsafe.html
Comment (McGiverGim) [inline src/tabs/failsafe.html:138]: I suppose this must be done depending on the version and not for all?

**Review by McGiverGim — CHANGES_REQUESTED** [review]

File: src/tabs/failsafe.html
Comment (haslinghuis) [reply src/tabs/failsafe.html]: Thanks. Fixed.

**Review by McGiverGim — APPROVED** [review]

**Review by KarateBrot — APPROVED** [review]

**Review by ctzsnooze — APPROVED** [review]

**Pull request merged**
